packages/terra-worklist-data-grid/src/orion-fusion-theme/ColumnHeaderCell.module.scss

```diff
@@ -11,7 +11,7 @@
 
     // Pinned Columns divider
     --terra-worklist-data-grid-pinned-columns-divider-background-color: #808080;
-    --terra-worklist-data-grid-pinned-columns-divider-box-shadow: 0;
+    --terra-worklist-data-grid-pinned-columns-divider-box-shadow: 4px -1px 6px #808080;
```

Comment:
If this matches the default theme, can't we just remove the declaration of this variable?